projects/demo/src/modules/components/input-number/input-number.template.html

```diff
@@ -259,6 +268,26 @@ <h3>There are also other components to input numbers:</h3>
             >
                 Show/hide decimal
             </ng-template>
+            <ng-template
+                i18n
+                documentationPropertyName="tuiTextfieldPrefix"
```

Comment:
Shouldn't we add those new props to the section with all other textfield controller props?

Reply:
will be moved to TextfieldControllerDocumentation after #3241